lift:
  publish_frequency: 10
